parts/k8s/kubernetesmasterresources.t

```diff
@@ -787,6 +802,9 @@
       },
       "location": "[variables('location')]",
       "name": "[concat(variables('masterVMNamePrefix'), copyIndex(variables('masterOffset')))]",
+      {{if .MasterProfile.HasAvailabilityZones}}
+      "zones": "[split(string(parameters('availabilityZones')[mod(copyIndex(variables('masterOffset')), length(parameters('availabilityZones')))]), ',')]",
```

Comment:
How does this handle copyIndex 0, for example:
```
        "AvailabilityZones": [
            "1",
            "2"
        ]
```
and cases where the AvailabilityZones array is not contiguous, which can happen.
```
        "AvailabilityZones": [
            "1",
            "3"
        ]
```